data/names/ru.json

```diff
@@ -8679,7 +9296,1005 @@
+      [ "Алан", "Alane" ],
+      [ "Алейда", "Aleida" ],
+      [ "Александр", "Alexander" ],
+      [ "Ален", "Alaine" ],
+      [ "Алеша", "Aleshia" ],
+      [ "Алида", "Alida" ],
+      [ "Алика", "Alica" ],
+      [ "Даймонд", "Diamond" ],
+      [ "Альберт", "Albert" ],
+      [ "Альма", "Alma" ],
+      [ "Ампаро", "Amparo" ],
+      [ "Ан", "An" ],
+      [ "Ангелена", "Angelena" ],
+      [ "Анджелес", "Angeles" ],
+      [ "Анджелин", "Angelyn" ],
+      [ "Анджело", "Angelo" ],
+      [ "Эндрю", "Andrew" ],
+      [ "Анжеле", "Angele" ],
+      [ "Анжелита", "Angelita" ],
+      [ "Анжелия", "Angelia" ],
+      [ "Аннали", "Annalee" ],
+      [ "Анналиса", "Annalisa" ],
+      [ "Аннамае", "Annamae" ],
+      [ "Аннемари", "Annemarie" ],
+      [ "Аннетта", "Annetta" ],
+      [ "Аннмари", "Annmarie" ],
+      [ "Антионетта", "Antionette" ],
+      [ "Антониетта", "Antonietta" ],
+      [ "Антонио", "Antonio" ],
+      [ "Аполония", "Apolonia" ],
+      [ "Арделл", "Ardell" ],
+      [ "Ализ", "Alease" ],
+      [ "Арлеан", "Arlean" ],
+      [ "Артур", "Arthur" ],
+      [ "Асусена", "Azucena" ],
+      [ "Аундреа", "Aundrea" ],
+      [ "Ауреа", "Aurea" ],
+      [ "Афтон", "Afton" ],
+      [ "Аяко", "Ayako" ],
+      [ "Базилия", "Basilia" ],
+      [ "Бао", "Bao" ],
+      [ "Бев", "Bev" ],
+      [ "Белл", "Bell" ],
+      [ "Белль", "Belle" ],
+      [ "Бёрилл", "Beryl" ],
+      [ "Бёрнадин", "Bernadine" ],
+      [ "Бёрнардин", "Bernardine" ],
+      [ "Бетанн", "Bethann" ],
+      [ "Бетси", "Betsey" ],
+      [ "Беттианн", "Bettyann" ],
+      [ "Беула", "Beula" ],
+      [ "Блейк", "Blake" ],
+      [ "Бобетт", "Bobette" ],
+      [ "Бок", "Bok" ],
+      [ "Бонг", "Bong" ],
+      [ "Боула", "Beaulah" ],
+      [ "Брайан", "Brian" ],
+      [ "Брайанна", "Bryanna" ],
+      [ "Бреана", "Breana" ],
+      [ "Бригида", "Brigida" ],
+      [ "Бриджитт", "Bridgett" ],
+      [ "Брэндон", "Brandon" ],
+      [ "Бэрри", "Berry" ],
+      [ "Вай", "Wai" ],
+      [ "Вайолет", "Violet" ],
+      [ "Вал", "Val" ],
+      [ "Валари", "Valarie" ],
+      [ "Вален", "Valene" ],
+      [ "Валентайн", "Valentine" ],
+      [ "Валерий", "Valeri" ],
+      [ "Ван", "Van" ],
+      [ "Ваня", "Vania" ],
+      [ "Варвар", "Barbar" ],
+      [ "Велва", "Velva" ],
+      [ "Велвет", "Velvet" ],
+      [ "Велла", "Vella" ],
+      [ "Вельда", "Velda" ],
+      [ "Венус", "Venus" ],
+      [ "Венис", "Venice" ],
+      [ "Вирджил", "Virgil" ],
+      [ "Верделл", "Verdell" ],
+      [ "Верлен", "Verlene" ],
+      [ "Вернелл", "Vernell" ],
+      [ "Вернон", "Vernon" ],
+      [ "Верси", "Versie" ],
+      [ "Спринг", "Spring" ],
+      [ "Бетел", "Bethel" ],
+      [ "Ви", "Vi" ],
+      [ "Вивиан", "Vivian" ],
+      [ "Виктор", "Victor" ],
+      [ "Вилла", "Willa" ],
+      [ "Виллен", "Willene" ],
+      [ "Виллена", "Willena" ],
+      [ "Виллетт", "Willette" ],
+      [ "Виллетта", "Willetta" ],
+      [ "Вилли", "Willie" ],
```

Comment:
These all should probably start with an `У`, rather than `В`. Names that start with *Will-* are generally transcribed with `Уилл-`. See: [`Умница Уилл Хантинг`](https://ru.wikipedia.org/wiki/%D0%A3%D0%BC%D0%BD%D0%B8%D1%86%D0%B0_%D0%A3%D0%B8%D0%BB%D0%BB_%D0%A5%D0%B0%D0%BD%D1%82%D0%B8%D0%BD%D0%B3), about the brilliant young man played by Matt Damon.